src/ledger/LedgerManagerImpl.cpp

```diff
@@ -328,6 +328,12 @@ LedgerManagerImpl::getDesiredLedgerVersion() const
     return mDesiredLedgerVersion;
 }
 
+void
+LedgerManagerImpl::setDesiredLedgerVersion(uint32_t desiredLedgerVersion)
```

Comment:
This is used in "Publish/catchup with too new protocol version" to test how nodes will behave when catching up to history with newer protocol version and what will happen when those nodes are themselves updated to the newest version.

I could remove checking what will happen after nodes are updated so this will not be needed.